kalite/distributed/templates/distributed/base.html

```diff
@@ -158,7 +158,7 @@
           CENTRAL_SERVER_HOST         : "{{ central_server_host }}",
           SECURESYNC_PROTOCOL         : "{{ securesync_protocol }}",
           CURRENT_LANGUAGE            : "{{ current_language }}",
-          CHANNEL                     : "{{ channel }}",
+          CHANNEL                     : "{{ channel }}"
```

Comment:
Ooooh... has anybody tested 0.14 on IE7 recently? This extra comma, in base.html, would likely break the whole site?   @EdDixon @MCGallaspy 

Do we even support IE7 anymore? Or only IE8?